.travis.yml

```diff
@@ -166,3 +168,4 @@ jobs:
       env:
         - NODE_VERSION: "chakracore/8"
         - VARIANT: "default"
+
```

Comment:
This line should better be removed, or it'd fail the EditorConfig test. I'd fix the issue in #735, mind to manually remove it first?

Reply:
Ah, right. I fix it.

Reply:
Actually, the `.update.sh` script keeps adding it back...we'll want to fix that

Reply:
Yes, I'd fix it in #735 ;)

Reply:
Didn't push it yet because I want #735 failed first, then fix it.